.github/workflows/reusable-build.yml

```diff
@@ -176,6 +176,17 @@ jobs:
       run: |
         cmake -G "Visual Studio 17 2022" -S external\ubpf -B external\ubpf\build
 
+    - name: Convert Markdown doc to HTML
+      # Convert markdown to HTML.
+      if: steps.skip_check.outputs.should_skip != 'true'
+      uses: jaywcjlove/markdown-to-html-cli@main
+      id: markdown_to_html
+      with:
+        source: ${{github.workspace}}/docs/TroubleShootingGuide.md
+        output: ${{github.workspace}}/docs/TroubleShootingGuide.html
+        github-corners: https://github.com/jaywcjlove/markdown-to-html-cli
+        favicon: data:image/svg+xml,<svg xmlns=%22http://www.w3.org/2000/svg%22 viewBox=%220 0 100 100%22><text y=%22.9em%22 font-size=%2290%22>🌐</text></svg>
```

Comment:
Per Anurag's comment, I think this file should be reverted, and the vcxproj file should be updated to do the conversion.

Reply:
My understanding is to revert the changes in 
- tools/nuget/ebpf-for-windows.nuspec.in

The reason is that the above places the TroubleShootingGuide.html in the release build, during msbuild, which is not found since the TroubleShootingGuide.html is building only in github actions in CICD pipeline.